x-pack/plugin/ccr/qa/multi-cluster/src/test/java/org/elasticsearch/xpack/ccr/FollowIndexIT.java

```diff
@@ -190,4 +195,40 @@ public void testChangeBackingIndexNameFails() throws Exception {
         assertThat(failure.getMessage(), containsString("a backing index name in the local and remote cluster must remain the same"));
     }
 
+    public void testFollowSearchableSnapshotsFails() throws Exception {
+        final String testPrefix = getTestName().toLowerCase(Locale.ROOT);
+
+        final String mountedIndex = "mounted-" + testPrefix;
+        if ("leader".equals(targetCluster)) {
+            final String repositoryPath = System.getProperty("tests.leader_cluster_repository_path") + '/' + testPrefix;
+            assertThat("Missing system property [tests.leader_cluster_repository_path]", repositoryPath, not(emptyOrNullString()));
```

Comment:
I think this can never be true? Did you mean to check that the system property is set?

Reply:
Thanks for catching this! 

Indeed I wanted to check the existence of the sysprop but I later added a suffix which made this assertion useless. Fixing the assertion allowed to catch a bug where the sysprop was not correctly passed everywhere so I pushed a2a5fdfb14426f9d923bb75eb242162176b02eaa to fix all of that.